packages/atlas-service/src/make-automation-agent-op-request.ts

```diff
@@ -1,3 +1,14 @@
+export type AtlasIndexStats = {
+  collName: string;
+  dbName: string;
+  indexName: string;
+  indexProperties: { label: string; properties: Record<string, unknown> }[];
+  indexType: { label: string };
```

Comment:
Weird that it is { label } and not immediately just the string. Trying to figure out where would be the ideal place to turn that string into an exact index type.